app/authentication/session_storage.py

```diff
@@ -88,6 +89,32 @@ def store_user_ik(user_ik):
         session[USER_IK] = user_ik
 
     @staticmethod
+    def store_survey_completed_metadata(tx_id, period_str, ru_ref):
+        """
+        Store survey_completed_metadata in the cookie for retrieval later
+        :param tx_id: the transaction id
```

Comment:
Not sure these `:param` descriptions add anything over the variable names; if they really do then consider just naming the variables more verbose instead.